.github/workflows/coverity-scan.yaml

```diff
@@ -0,0 +1,45 @@
+# Tests in this workflow are run periodically.
+# They are generally very expensive and not on the critical path
+# for development
+# scheduled workflows only run in the default branch
+
```

Comment:
Let's not document what scheduled tests are and avoid arguing regarding "critical path for development":

```suggestion
```

If you would like to explain why Coverity Scan tests should be executed on a schedule, then let's do that:

```suggestion
# Coverity Scan service terms limit analysis requests frequency, 
# and the service runs analysis in the background. Thus, we submit
# default branch analysis requests on a schedule rather than testing PRs.

```